src/Service/Kinesis/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: Added StreamARN parameter for Kinesis Data Streams APIs. Added a new opaque pagination token for ListStreams. SDKs will auto-generate Account Endpoint when accessing Kinesis Data Streams.
+- AWS api-change: Amazon Kinesis Data Streams now supports on demand streams.
```

Comment:
```suggestion
- AWS api-change: Added StreamARN parameter for Kinesis Data Streams APIs. Added a new opaque pagination token for ListStreams. SDKs will auto-generate Account Endpoint when accessing Kinesis Data Streams.
- BC BREAK: `listStreams` now yield over stream names AND descriptions
```

@Nyholm From a DX point of view, I wonder if yielding over both make sense :thinking: 

See documentation of operation https://docs.aws.amazon.com/kinesis/latest/APIReference/API_ListStreams.html